test_runner/fixtures/utils.py

```diff
@@ -647,6 +649,24 @@ def allpairs_versions():
     return {"argnames": "combination", "argvalues": tuple(argvalues), "ids": ids}
 
 
+def size_to_bytes(hr_size: str) -> int:
+    """
+    Gets human-readable size from postgresql.conf (e.g. 512kB, 10MB)
+    returns size in bytes
+    """
+    M_PREFIXES = {"": 1, "k": 1024, "M": 1024**2, "G": 1024**3, "T": 1024**4, "P": 1024**5}
+    match = re.search(r"^\'?(\d+)\s*([kMGTP]?)(B?)\'?$", hr_size)
+    assert match is not None, f"'{hr_size}' is not well-formatted human-readable size"
+    number, m_prefix, b = match.groups()
+
+    assert not (not b and m_prefix), f"'{hr_size}' is not well-formatted human-readable size"
```

Comment:
Probably we can simplify it a bit:

```suggestion
    assert b and not m_prefix, f"'{hr_size}' is not well-formatted human-readable size"
```

Reply:
This code was removed